src/Core/Prefab/GlobeView.js

```diff
@@ -229,11 +230,14 @@ function GlobeView(viewerDiv, coordCarto, options) {
     this._renderState = RendererConstant.FINAL;
 
     this.preRender = () => {
-        var len = this.camera.position().length();
-        var lim = size * 1.1;
+        const v = new THREE.Vector3();
+        v.setFromMatrixPosition(wgs84TileLayer.object.matrixWorld);
+        var len = v.distanceTo(this.camera.position());
+        v.setFromMatrixScale(wgs84TileLayer.object.matrixWorld);
+        var lim = v.x * size * 1.1;
```

Comment:
The previous code did this:
```js
const size = ellipsoidSizes().x;
[...]
var lim = size * 1.1;
```
So the new code does the same computation, but it doesn't assume that:
1) the globe is in (0,0,0) (hence the need to use `distanceTo` instead of `position.length()`)
2) the ellipsoidSize is fixed (that's why we use `setFromMatrixScale`)